src/input/mod.rs

```diff
@@ -1127,6 +1127,21 @@ impl State {
                 // FIXME: granular
                 self.niri.queue_redraw_all();
             }
+            Action::MoveWorkspaceToIndex(new_idx) => {
+                self.niri.layout.move_workspace_to_idx(None, new_idx);
+            }
+            Action::MoveWorkspaceToIndexByRef {
+                new_idx: index,
```

Comment:
Sounds like this doesn't need to be renamed

Reply:
What do you mean by that? There is now a `MoveWorkspaceToIndex` and a `MoveWorkspaceToIndexByRef`. The same way as I did for the ToMonitor actions.

Reply:
I mean that instead of `new_idx: index` just have `new_idx`

Reply:
Ah yea that makes sense, that's done now.